README-SW.md

```diff
@@ -0,0 +1,232 @@
+git checkout -b update-readme-file<!-- Do not translate this -->
```

Comment:
hello,
on this line, i guess `git checkout -b update-readme-file` was added by mistake and should be **removed** you can work on that buddy.